messaging/src/test/java/org/axonframework/messaging/eventhandling/processing/streaming/pooled/PooledStreamingEventProcessorTest.java

```diff
@@ -1216,85 +1359,101 @@ void startingAfterShutdownLetsProcessorProceed() {
         }
 
         @Test
-        void supportReset() {
-//            when(stubEventHandler.supportsReset()).thenReturn(true);
-
+        void supportsResetReturnsTrueWhenComponentSupportsReset() {
             assertTrue(testSubject.supportsReset());
-
-//            when(stubEventHandler.supportsReset()).thenReturn(false);
-
-            assertFalse(testSubject.supportsReset());
         }
 
         @Test
         void resetTokensFailsIfTheProcessorIsStillRunning() {
             startEventProcessor();
 
-            assertThrows(IllegalStateException.class, () -> testSubject.resetTokens());
+            var thrown = assertThrows(IllegalStateException.class, () -> joinAndUnwrap(testSubject.resetTokens()));
+            assertEquals("The Processor must be shut down before triggering a reset.", thrown.getMessage());
         }
 
         @Test
         void resetTokens() {
```

Comment:
Looking at this test...isn't this a duplicate of the `resetTokensFromDefinedPosition()` test method?

Reply:
Those assertions pass just in the resetTokens:
```
          assertEquals(expectedToken, token0);
          assertEquals(expectedToken, token1);
```

I will check why later. I'm almost sure both tests were in the original test class.